core/tests/webdriverio_desktop/playVoiceovers.js

```diff
@@ -53,7 +53,9 @@ describe('Voiceover player', function () {
     );
     await explorationEditorMainTab.setInteraction('EndExploration');
     await explorationEditorPage.navigateToTranslationTab();
+    await browser.pause(10000);
```

Comment:
Please don't use browser.pause, as it's a common source of flakes

Reply:
done!